pkg/controller/types/networkpolicy.go

```diff
@@ -116,4 +116,7 @@ type NetworkPolicy struct {
 	// TierPriority represents the priority of the Tier associated with this Network
 	// Policy.
 	TierPriority *int32
+	// Description user can store extra information to describe the intention of
+	// those policies.
+	Description string
```

Comment:
i think this may also not be needed at this time

Reply:
Yeah sure.